doi/README.md

```diff
@@ -24,60 +24,55 @@ See <a href="https://github.com/opencadc/reg/tree/master/cadc-registry">cadc-reg
 The doi.properties configures the DataCite service used to register new DOI's.
 
 ```
-# Vault Service resourceID
-ca.nrc.cadc.doi.vaultResourceID = {vault service resourceID}
-
-# Group Management Service (GMS) resourceID
-ca.nrc.cadc.doi.gmsResourceID = {GMS service resourceID}
-
-# Path in vault to the parent folder containing the DOI's.
-ca.nrc.cadc.doi.parentPath = {DOI parent folder path}
+# VOSpace uri to the parent DOI folder.
+ca.nrc.cadc.doi.vospaceParentUri = {parent folder URI}
 
 # Prefix to the DOI metadata file
-ca.nrc.cadc.doi.metadataFilePrefix = {file prefix}
+ca.nrc.cadc.doi.metaDataPrefix = {metadata file prefix}
+
+# Prefix to the DOI GMS Group name
+ca.nrc.cadc.doi.groupPrefix = {group prefix}
  
 # DOI landing page url
 ca.nrc.cadc.doi.landingUrl = {landing page url}
 
 # DataCite MDS REST endpoint
 ca.nrc.cadc.doi.datacite.mdsUrl = {MDS url}
 
-# DataCite account prefix
-ca.nrc.cadc.doi.datacite.accountPrefix = {account prefix}
-
 # DataCite account username
 ca.nrc.cadc.doi.datacite.username = {username}
 
 # DataCite account password
 ca.nrc.cadc.doi.datacite.password = {password}
-```
 
-_vaultResourceID_ the resourceID to the vault service used to store the DOI metadata and files.
+# DataCite account prefix
+ca.nrc.cadc.doi.datacite.accountPrefix = {account prefix}
+```
 
-_gmsResourceID_ the resourceID to the GMS service used for authentication and authorization.
+_parentUri_ is the URI to the DOI parent folder in the VOSpace service.
 
-_parentPath_ is the path in the vault service to the DOI parent folder.
+_metaDataPrefix_ is the prefix prepended to the DOI name used to create the file for the DOI specific metadata stored in VOSpace.
 
-_metadataFilePrefix_ is the prefix prepended to the DOI name to create the filename for the DOI specific metadata stored in VOSpace.
+_groupPrefix_ is the prefix prepended to the DOI name to create the group name for the DOI.
 
 _landingUrl_ is the base URL used to compose URLs to individual DOI's.
 
-_mdsUrl_ is the URL to the Datacite MDS rest endpoint used to create and update DOI's.
-
-_accountPrefix_ is the registered prefix for the DataCite account.
+_mdsUrl_ is the URL to the DataCite MDS rest endpoint used to create and update DOI's.
 
 _username_ is the DataCite account username.
 
 _password_ is the DataCite account password.
 
+_accountPrefix_ is the registered prefix for a DataCite account.
+
 **For developer testing only:**
 ```
-# (optional) Create a random DOI name for testing
-ca.nrc.cadc.doi.test.randomName = {true|false}
-
-# (optional) Group URI for a group that has read/write permissions to test DOI's
-ca.nrc.cadc.doi.test.groupUri = {group URI}
+# (optional) Create a random DOI ID for testing
+ca.nrc.cadc.doi.randomTestID = {true|false}
```

Comment:
Is this really necessary? Can just the prefix and different test directory be enough? Under what circumstances would int testing interfere with production and what would be the consequences? 
If you can get rid of this then you can remove traces of test code in the service code as well.

Reply:
Using the standard ID of 2 digit year plus a sequential 4 digit number means the ID could repeat in int-tests. The old DOIs would have to be cleaned up in the test DataCite service, and also the groups created for the DOIs would have to be cleaned up as well. Using a random ID does create test DOIs and groups, but it will work every time, so I think this is the right approach.